<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Rename</class>
 <widget class="QWidget" name="Rename">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>344</width>
    <height>109</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Rename Sensor</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="serialNumberLabel">
       <property name="toolTip">
        <string>Serial Number</string>
       </property>
       <property name="text">
        <string>Serial Number:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="sensorNameLabel">
       <property name="toolTip">
        <string>Sensor Name</string>
       </property>
       <property name="text">
        <string>Sensor Name:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="sensorNameLineEdit">
       <property name="toolTip">
        <string>Enter sensor name metadata</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="serialNumberLabel2">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalButtonlLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="renameButton">
       <property name="toolTip">
        <string>Import and Next</string>
       </property>
       <property name="text">
        <string>Rename</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="toolTip">
        <string>Cancel and Close</string>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
